psd-web/app/helpers/investigations/display_text_helper.rb

```diff
@@ -103,10 +103,12 @@ def correspondence_should_be_hidden(result, source, investigation)
   end
 
   # rubocop:disable Rails/OutputSafety
-  def investigation_assignee(investigation, classes = "")
-    out = [investigation.assignable ? investigation.assignable.name.to_s : tag.div("Unassigned", class: classes)]
-    out << tag.div(investigation.assignable.organisation.name, class: classes) if investigation.assignable&.organisation.present?
-    out.join.html_safe
+  def investigation_assignee(investigation)
+    out = [investigation.assignable ? h(investigation.assignable.name.to_s) : h("Unassigned")]
```

Comment:
```suggestion
    out = [investigation.assignable ? h(investigation.assignable.name.to_s) : "Unassigned"]
```
no need to escape "Unassigned" it doesn't contain malicious markup